pdk/1.1/modules/ROOT/pages/policies-pdk-configure-features-timer.adoc

```diff
@@ -0,0 +1,161 @@
+= Configuring Delayed, Periodic, and Synchronous Functions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+PDK provides the `Clock` injectable to build a timer to enable delayed, periodic, and synchronous functions. Each policy only supports one timer.
```

Comment:
Just checking - PDK is an acceptable initialism? Should we expand on first mention?

Reply:
Can you flip this to put the customer value first?

Reply:
Something like "Use the Clock injectable to build a timer to...."